features/ios/support/utils.rb

```diff
@@ -0,0 +1,91 @@
+def turn_off_bluetooth
+	# not yet implemented
+end
+
+def give_location_permission
+	# after launching the application there might be pop-up for permission location access
+	if device_agent.springboard_alert_visible?
+		puts "- Needs to give permission, let's give a permission"
+
+		begin
+			device_agent.dismiss_springboard_alert("Always Allow")
+		rescue
+			device_agent.dismiss_springboard_alert("Allow")
+		end
+	end
+end
```

Comment:
SpringBoard alerts are automatically dismissed.  If they are not automatically dismissed there is an error in DeviceAgent.

Remove these 2 methods: `allow_notification` and `give_location_permission`.